src/libs/portfolio/pagination.ts

```diff
@@ -8,6 +8,30 @@ export function paginate(input: any[], limit: number): any[][] {
   return pages
 }
 
-export async function flattenResults(everything: Promise<any[]>[]): Promise<any[]> {
-  return Promise.all(everything).then((results) => results.flat())
+export function flattenResults(everything: Promise<any[]>[]): Promise<any[]> {
```

Comment:
we can add types to this to make this code safer.

And to the function that returns the results that we pass into here